lib/internal/v8_prof_polyfill.js

```diff
@@ -100,3 +101,30 @@ function versionCheck() {
     }
   }
 }
+
+function macCppfiltNm(out) {
+  // Re-grouped copy-paste from `tickprocessor.js`
+  const FUNC_RE = /^([0-9a-fA-F]{8,16} [iItT] )(.*)$/gm;
```

Comment:
It is a copy-paste from V8, but I agree with you. It doesn't matter that much in the end, though.